themes/bootstrap3/templates/RecordTab/holdingsils/electronic.phtml

```diff
@@ -3,12 +3,20 @@
   <?php foreach ($this->holdings as $item): ?>
     <tr>
       <td class="fullLocation">
-        <?php $locationTextEsc = $this->transEscWithPrefix('location_', $item['location']); ?>
-        <?php if ($item['locationhref'] ?? false): ?>
-          <a href="<?=$this->escapeHtmlAttr($item['locationhref'])?>" target="_blank"><?=$locationTextEsc?></a>
-        <?php else: ?>
-          <?=$locationTextEsc?>
+        <?php if (!is_array($item['location'])): ?>
+          <?php $item['location'] = [$item['location']]; ?>
         <?php endif; ?>
+        <?php foreach ($item['location'] as $index => $location): ?>
```

Comment:
Thanks for the improvements, @hajoseng. You can further simplify this by removing the `is_array` check and just changing your loop to `<?php foreach ((array)$item['location'] as $index => $location): ?>` -- this does exactly the same thing through type-casting and saves you three lines of code. :-)